models/newsModels.js

```diff
@@ -9,29 +9,41 @@ exports.fetchTopics = () => {
 };
 
 
-exports.fetchArticles = (sort_by, order) => {
+exports.fetchArticles = (sort_by = 'created_at', order = 'DESC', topic) => {
+  const validSortColumns = ['title', 'topic', 'author', 'body', 'created_at', 'votes', 'article_img_url'];
+  const validTopics = ['mitch', 'cats'];
```

Comment:
You may find this style of greenlist could be brittle and cause issues - for example the test data and the development data have different topics! Is there a way to check against the database so it's a more dynamic check?

Reply:
Thanks for feedback Carrie. I have solution but don't know if it's correct, if I am allowed to do what I did. Would you like to call me before I upload it? Or should I just send another pr for review?